pkg/v1/tkg/web/src/app/views/landing/aws-wizard/vpc-step/vpc-step.fieldmapping.ts

```diff
@@ -1,9 +1,10 @@
+// App imports
 import { AwsField, VpcType } from '../aws-wizard.constants';
 import { StepMapping } from '../../wizard/shared/field-mapping/FieldMapping';
 
 export const AwsVpcStepMapping: StepMapping = {
     fieldMappings: [
-        { name: AwsField.VPC_TYPE, required: true, defaultValue: VpcType.NEW },
+        { name: AwsField.VPC_TYPE, required: true, defaultValue: VpcType.EXISTING },
```

Comment:
Is this change in default intentional? Sorry if I'm forgetting a conversation about this. I feel like we may have talked about it but not remembering why.

Reply:
Yes, this was intentional; part of the feedback on how folks use the UI